README.md

```diff
@@ -214,6 +214,8 @@ environment variable:
   as out-of-bounds accesses) first.  Setting this flag means Miri can miss bugs
   in your program.  However, this can also help to make Miri run faster.  Using
   this flag is **unsound**.
+* `-Zmiri-disable-abi-check` disables checking [function ABI]. Using this flag
```

Comment:
This list is intended to be alphabetically sorted... I realize "disable-isolation" is already at the wrong spot, but if you could add the new flag at the right spot, that would be great. :)